### PR TITLE
Fix Bottom Navbar & Top and Bottom Navbar On Same Page 

### PR DESCRIPTION
**Pull request by jonny827**
This fixes Issue #1174.

Thank you @isometriks .

### CODE REVIEW

**Comment by jonny827** [comment]
Shall I rebase and resubmit this PR @isometriks

**Comment by isometriks** [comment]
Nope, should be okay. I think I tried to comment on this earlier but it didn't go through. I think it should probably be `navbarId` instead to match the rest of the variables. Don't worry about squashing anymore, I figure out how to do that now, I didn't see the option, but I have to click "Merge pull request" first before I saw the option when I merged your last PR.

**Comment by jonny827** [comment]
Thank you for your reply. No worries. GitHub went down the other night on night while trying to push.  Squashing from your side, as I mentioned in the other PR with that link it is a very new feature. I think it came out late last month according to the date of the post. I agree on the variables name. I did not look at consistency within this template. The variables in the other template used snake case.

**Comment by jonny827** [comment]
@isometriks Thank you for helping me fix Issue #1174.   I have adjusted the variable name and the corresponding documentation.

**Comment by jonny827** [comment]
@isometriks  When you have a moment can you please merge this.

**Comment by jonny827** [comment]
@phiamo   Can you please merge this, it is ready and I think @isometriks might be pretty busy as he has not gotten back to me.  Thank you in advance.

**Comment by isometriks** [comment]
Sorry @jonny827 getting back to reviewing some stuff now. Do you think we can just use `id` instead of passing another `navbarId` variable?

**Comment by jonny827** [comment]
@isometriks  No worries. Thank you for getting back to me,  I can updated the commit to use variable id passed in versus navbarId.   Anything else you see or can we commit this?  Thank you.

File: Resources/views/Navbar/navbar.html.twig [hunk 1]
```diff
@@ -14,7 +14,7 @@
         </div>
         {% endblock header %}
         {% block menu_container %}
-        <div class="collapse navbar-collapse navbar-responsive-collapse">
+        <div class="collapse navbar-collapse navbar-responsive-collapse-{% if id|default(false) %}{{ id }}{% endif %}">
```
Comment (isometriks) [inline Resources/views/Navbar/navbar.html.twig]: Should be:

`navbar-responsive-collapse{% if id|default(false) %}-{{ id }}{% endif %}`

For both of them. Don't put the hyphen at the end unless there's an ID

**Comment by jonny827** [comment]
@isometriks   Thank you for catching the end dash. It definitely needed to be inside the conditional.  It should be ready to go now.  I appreciate your help on getting this part of the project!

**Comment by isometriks** [comment]
:+1: Thanks @jonny827

**Comment by jonny827** [comment]
You are welcome! Thank you ! @isometriks .  I really appreciate your assistance in getting this PR committed.

**Pull request merged**